### DISCUSSION TITLE
Welcome to CrackPy Discussions!

### DISCUSSION THREAD

**Discussion by melc-da** [dlr-wf/crackpy#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MGeorg93** [comment]
Hello everyone! Mike here I am conducting research on composite materials and I was looking for automated tools to monitor crack propagation according to ASTM D5528 for Fracture Toughness Mode I. Has anyone tried to use CrackPy to monitor the crack propagation from video material? Thank you in advance!

**Reply by brei-er** [reply]
CrackPy is designed to work with Digital Image Correlation (DIC) data. You only need displacement and strain data, regardless of the data source. A video camera could work if the image quality is good enough for DIC evaluation. However, I always recommend the use of global shutter cameras.
There are even some open source tools for 2D DIC evaluation. Just give it a try with your data.
